src/frontend/Shared/components/FriendList.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { connect} from 'react-redux';
+import { increment } from '../actions';
+import {GlobalState} from "../reducers";
+
+interface FriendListProps {
+    isClose? : false;
+}
+
+class FriendList extends React.Component<FriendListProps, {}> {
+    constructor(props: any) {
+        super(props);
+    }
+
+    collapseRight = () => {
+        let element = document.getElementById("right-panel");
```

Comment:
I don't think the use of ID is good here. It add assumption to HTML file that there is no other element has 'right-panel' as its ID.

Reply:
right I agree, it will be better to use classname, but since the problem will have only this right panel, let us keep this here

Reply:
But the issue is you have many IDs in your components, which might cause unexpected errors in the future